matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/sync/UserAccountDataSyncHandler.kt

```diff
@@ -56,21 +63,23 @@ internal class UserAccountDataSyncHandler @Inject constructor(
 
     fun handle(realm: Realm, accountData: UserAccountDataSync?) {
         accountData?.list?.forEach {
-            when (it) {
-                is UserAccountDataDirectMessages -> handleDirectChatRooms(realm, it)
-                is UserAccountDataPushRules      -> handlePushRules(realm, it)
-                is UserAccountDataIgnoredUsers   -> handleIgnoredUsers(realm, it)
-                is UserAccountDataBreadcrumbs    -> handleBreadcrumbs(realm, it)
-                is UserAccountDataFallback       -> Timber.d("Receive account data of unhandled type ${it.type}")
```

Comment:
Maybe keep this log in the else block?

Reply:
The type as already been handled as a generic account data type. 
It's not really a warning, it is not 'unhandled', there is just no specific processing

Reply:
It could help the developers (AKA us) to detect that some treatment is missing/may be handled